### PR TITLE
#167313374 Search and Filter UI design

### PR DESCRIPTION
**Pull request by minega25**
#### What does this PR do?
User interface design for search and filter components
#### Description of Task to be completed?
Build search component user interface design.
#### How should this be manually tested?
git fetch
git checkout ft-search-filter-167313374
npm run devserver

Navigate to /search
Navigate to /search/detail
#### What are the relevant pivotal tracker stories?
#167313374
#### Screenshots (if appropriate)
Screen Shot 2019-10-16 at 10 00 14
Screen Shot 2019-10-16 at 09 59 53
Screen Shot 2019-10-16 at 10 00 04
Screen Shot 2019-10-16 at 09 59 47
Screen Shot 2019-10-16 at 09 59 12

### CODE REVIEW

File: src/components/search/search.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```
Comment (habinezadalvan) [inline src/components/search/search.js]: fix the linting issue

**Review by habinezadalvan — APPROVED** [review]
LGTM, nice work

File: src/components/search/search.js (hunk 1 above)
Comment (minega25) [reply src/components/search/search.js]: Fixed this issue

**Review by nkalyesubula — CHANGES_REQUESTED** [review]
Nice work PATRICK. A few observations though. You could remove the plus on the follow button, there is also much HTML code in the class-based component and some are duplicated, you could create static functional components and use them in the class-based component. Also, make the user story clickable

**Comment by minega25** [comment]
> Nice work PATRICK. A few observations though. You could remove the plus on the follow button, there is also much HTML code in the class-based component and some are duplicated, you could create static functional components and use them in the class-based component. Also, make the user story clickable

@nkalyesubula I have worked on this.

**Review by nkalyesubula — APPROVED** [review]

**Review by Anguandia — APPROVED** [review]
Very beautiful UI, I like it and it looks professionally done. If, however, you may consider changing anything, we could try to get the page colors a little more in harmony; green and blue are not so fine a combination, but +1000

File: src/components/common/articleCard/ratingComponent/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Rating from 'react-rating';
+import style from './ratingComponent.scss';
+
+const theme = {
+  color: '#ffd700',
+  border: '1px solid #fff',
+  fontSize: '13px',
+};
+export default class RatingArticleCardComponent extends Component {
```
Comment (SokoPaulSokool) [inline src/components/common/articleCard/ratingComponent/index.js]: Rating component should be put directly in the common since it will be reused

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]

File: src/components/common/articleCard/ratingComponent/index.js (hunk 2 above)
Comment (minega25) [reply src/components/common/articleCard/ratingComponent/index.js]: Alright.

Comment (minega25) [reply src/components/common/articleCard/ratingComponent/index.js]: This has been fixed in the landing page PR. We are using the same articleCard component.

File: src/components/search/search.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                  coverImage={article && article.images && article.images[0]}
+                  description={article && article.description}
+                  title={article && article.title}
+                  views={article && article.views}
+                  readTime={article && article.readtime}
+                  userIcon={article && article.userImage}
+                  username={article && article.username}
+                  createdTime={article && article.timeCreated}
+                />
+              </div>
```
Comment (SokoPaulSokool) [inline src/components/search/search.js]: You should use the listOfArticles component

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]

**Pull request merged**